packages/camel-catalog/kaoto-camel-catalog-maven-plugin/src/main/java/io/kaoto/camelcatalog/KaotoCamelCatalogMojo.java

```diff
@@ -111,7 +112,7 @@ public void execute() {
         var index = new Index();
         var yamlDslSchemaProcessor = processCamelSchema(path, index);
         processK8sSchema(path, index);
-        processCatalog(yamlDslSchemaProcessor, index);
+        processCatalog(yamlDslSchemaProcessor, index, path);
```

Comment:
Very minor thing: since the other methods are using `path` & `index`, in that order, should we aim for the same in this `processCatalog` method, consistency-wise?

For instance:
```java
processCatalog(yamlDslSchemaProcessor, path, index) 
```